Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md

```diff
@@ -579,7 +579,7 @@ Before we describe the usable identifiers, let's quickly discuss how they can be
 - You can store something in the Keychain to identify the application's instance. To make sure that this data is not backed up, use `kSecAttrAccessibleWhenPasscodeSetThisDeviceOnly` (if you want to secure the data and properly enforce a passcode or touch-id requirement), `kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly`, or `kSecAttrAccessibleWhenUnlockedThisDeviceOnly`.
```

Comment:
```suggestion
- You can store something in the Keychain to identify the application's instance. To make sure that this data is not backed up, use `kSecAttrAccessibleWhenPasscodeSetThisDeviceOnly` (if you want to secure the data and properly enforce a passcode or Touch ID requirement), `kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly`, or `kSecAttrAccessibleWhenUnlockedThisDeviceOnly`.
```

Reply:
Please revise the consistency of Touch ID and Face ID in this file and in 0x6f at least. Thank you :)

Reply:
Done :D thanks!